src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -22,24 +22,16 @@ import Text from '../../components/Text';
 import * as BankAccounts from '../../libs/actions/BankAccounts';
 import ONYXKEYS from '../../ONYXKEYS';
 import compose from '../../libs/compose';
-import * as ReimbursementAccountUtils from '../../libs/ReimbursementAccountUtils';
-import ReimbursementAccountForm from './ReimbursementAccountForm';
-import reimbursementAccountPropTypes from './reimbursementAccountPropTypes';
 import Section from '../../components/Section';
 import * as ValidationUtils from '../../libs/ValidationUtils';
 import * as Illustrations from '../../components/Icon/Illustrations';
 import getPlaidDesktopMessage from '../../libs/getPlaidDesktopMessage';
 import CONFIG from '../../CONFIG';
 import ROUTES from '../../ROUTES';
 import Button from '../../components/Button';
-import FormScrollView from '../../components/FormScrollView';
-import FormAlertWithSubmitButton from '../../components/FormAlertWithSubmitButton';
+import Form from '../../components/Form';
 
 const propTypes = {
-    /** Bank account currently in setup */
-    // eslint-disable-next-line react/no-unused-prop-types
-    reimbursementAccount: reimbursementAccountPropTypes.isRequired,
```

Comment:
qq: Does removing reading `reimbursementAccount` have an impact on this? I thought these values are used as default values in the form?

Reply:
@mananjadhav I thought `shouldSaveDraft` prop will handle this. but I will take reference in a day or 2 from the other form refactor tasks that removing onyx will break the flow or not.

Reply:
@luacmartins I have a doubt here, we are using REIMBURSEMENT_ACCOUNT key for storing all the reimbursement data(bank account, company, requestor step, etc..), so from now on how we are going to handle this data under different ONYX form keys, or should we have to use the same form key for all the reimbursement form steps (CompanyStep, WorkspaceBankAccountPage, BankAccountStep)?

Reply:
I think we should continue to use the same key for all steps